packages/grafana-llm-app/src/components/AppConfig/LLMConfig.tsx

```diff
@@ -234,21 +291,33 @@ export function LLMConfig({
               </Card.Figure>
             </Card>
           </div>
+          </div>
         )}
+        <div className={s.providerSection}>
+          <div className={s.sectionHeader}>
+            <h4>Bring Your Own Service {(llmOption === 'openai' || llmOption === 'anthropic' || llmOption === 'custom') && <Badge text="Currently Active" color="green" />}</h4>
+          </div>
```

Comment:
this looks a bit unusual when `allowGrafanaManagedLLM` is falsey because it says 'Currently Active' but is the only option. It made me wonder what the alternatives are 😅 I wonder if we could also gate it behind the same variable?